docs/segments/manage_segments.rst

```diff
@@ -3,4 +3,493 @@
+
+.. image:: images/points-trigger.png
+    :alt: Screenshot showing Points trigger.
+
+1. In the **Points** section of the platform, select** Manage Triggers**.
+
+2. Click **+New**.
+
+3. Give your trigger a **Name** and **Description**.
+
+4. Enter the **Minimum number of Points** for adding a Contact to your Segment.
+
+5. Decide if you’d like to add all Contacts with at least that number of Points to the Segment:
+ 
+  * If you only want to add new Contacts who reach the threshold to this Segment, select **No** (default).
+  
+  * To add all existing Contacts with at least a certain number of Points to the Segment, toggle the switch to **Yes**.
+
+6. Click **Events**.
+
+7. Click the menu for **Add an event** and select **Modify Contact’s Segments**.
+
+8.  Add a **Title** for the event and a **Description** (optional).
+
+9.  Select the Segments you’d like to add to or remove the Contact from.
+
+10. Click **Add**.
+
+11. Click **Save & Close**.
+
+This accomplishes the same thing as creating a Dynamic Segment with a filter for Points, the operator ``greater than (or equal to)``, and the minimum number of Points. The difference is if you only want to add Contacts to a Segment who reach the point value after creating this trigger to a Segment, you can.
+
+.. vale off
+
+CSV upload
+===========
+
+.. vale on
+
+If you have a list outside of Mautic, saved as a UTF-8 encoded CSV file, you can upload the list directly into a Mautic Segment.
+
+1. Click **Contact**.
+
+2. Click the menu in the upper-right hand corner and select **Import**.
+
+3. Ensure the file is in a UTF-8 CSV format. Select your file then click **Upload**.
+
+4. At the top of the next screen, select the Segment you want to add the Contacts to.
+
+5. Map the appropriate fields from the file. Remember to always map the unique identifier, so you don’t create duplicates.
+
+6. Click **Import**.
+
+If your file is larger than 1,000 rows, the system changes screens and informs you once the file has uploaded. After the upload is complete, click the Segments section of the platform to see the added Contacts.
+
+.. vale off
+
+Dynamic Segments
+****************
+
+.. vale on
+
+.. image:: images/filtering.png
+    :alt: Screenshot showing Segment filters.
+
+Mautic moves Contacts into and out of dynamic Segments based on the filters applied to the Segment. As the data associated with the Contact updates, including Company associations and behaviors, Mautic updates Segment membership.
+
+Segment filters
+===============
+
+.. image:: images/filter-list.png
+    :width: 400
+    :alt: Screenshot showing Segment Filters List.
+
+1. Create a new Segment by clicking the **+New** button.
+
+2. In the **Details** tab, add a **Name**, **Public name** and **Description** to your Segment.
+
+3. Click the **Filters** tab to add filters.
+
+4. Click the **Choose one**… menu and search for the field you’d like to Segment by. Listed below are three types of fields:
+      
+  * Contact fields
```

Comment:
Same issue here with the indentation - maybe it is 4 rather than 3 spaces?

Reply:
I tried 4 spaces and it is still same